app/src/main/java/com/sopt/now/Friend/Friend.kt

```diff
@@ -1,9 +1,18 @@
 package com.sopt.now.Friend
 
-import androidx.annotation.DrawableRes
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
 
+@Serializable
 data class Friend(
-    @DrawableRes val profileImage :Int,
-    val nickname : String,
-    val mbti : String
+    @SerialName("id")
+    val id: Int,
+    @SerialName("email")
+    val email: String,
+    @SerialName("first_name")
+    val first_name: String,
```

Comment:
맞아요! 변수명은 카멜케이스로 모두 적용해주세요

```suggestion
    @SerialName("first_name")
    val firstName: String,
```